scripts/helper_functions.sh

```diff
@@ -2,6 +2,14 @@
 
 # Common functions definitions
 
+function wait_for_apt
```

Comment:
Can we rename the function to wait for apt to finish. Also can we add a comment on top of function explaining in detail the issue that this function is addressing ?